mlflow/server/js/src/components/ExperimentListView.js

```diff
@@ -57,19 +118,36 @@ export class ExperimentListView extends Component {
                 className = `${className} active-experiment-list-item`;
               }
               return (
-                <Link
-                  style={{ textDecoration: 'none', color: 'unset' }}
-                  key={e.getExperimentId()}
-                  to={Routes.getExperimentPageRoute(e.getExperimentId())}
-                  onClick={active ? ev => ev.preventDefault() : ev => ev}
-                >
-                  <div
-                    className={className}
-                    title={e.getName()}
+                <div key={e.getExperimentId()} title={e.getName()}
+                  className={`header-container ${className}`}>
+                  <Link
+                    style={{ textDecoration: 'none', color: 'unset', width: '80%'}}
+                    to={Routes.getExperimentPageRoute(e.getExperimentId())}
+                    onClick={active ? ev => ev.preventDefault() : ev => ev}
+                  >
+                    <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
+                      {e.getName()}
+                    </div>
+                  {/* Edit/Rename Experiment Option */}
+                  </Link>
+                  <a
+                    onClick={this.onRenameExperiment}
+                    data-experimentid={e.getExperimentId()}
+                    data-experimentname={e.getName()}
+                    style={{ marginRight: 10 }}
+                  >
+                    <Icon type="edit" />
+                  </a>
+                  {/* Delete Experiment option */}
+                  <a
+                    onClick={this.onDeleteExperiment}
+                    disabled={active}
+                    data-experimentid={e.getExperimentId()}
+                    data-experimentname={e.getName()}
                   >
-                    {e.getName()}
-                  </div>
-                </Link>
+                    <Icon type="delete" />
```

Comment:
@ggliem 

The image below is how the delete-tag button looks like on the run view page. I think the delete-experiment button should use the same icon.

<img width="423" alt="Screen Shot 2020-01-28 at 16 56 05" src="https://user-images.githubusercontent.com/17039389/73245461-9c178e00-41ef-11ea-8190-7ce9cf768d78.png">


Code

https://github.com/mlflow/mlflow/blob/690c7e933f4c1007f477fa2b240631a9f8f515cf/mlflow/server/js/src/components/tables/EditableFormTable.js#L119-L121

Reply:
btw this feature looks awesome!

Reply:
Sure, I can change that (I actually used the same implementation in the beginning)